src/data/gedi_database.py

```diff
@@ -118,6 +123,7 @@ def query(
         limit: int = None,
         use_geopandas: bool = False,
         force: bool = False,
+        order_by: List[str] = [],
```

Comment:
I know the whole function doesn't have a docstring ... but if you wouldn't mind writing one it does feel like it would be helpful to specify the order_by syntax for callers.